include/picongpu/fields/background/templates/twtstight/BField.tpp

```diff
@@ -205,29 +205,13 @@ namespace picongpu
                 auto const beta0 = float_T(beta_0);
                 /* If phi < 0 the formulas below are not directly applicable.
                  * Instead phi is taken positive, but the entire pulse rotated by 180 deg around the
-                 * z-axis of the coordinate system in this function.
-                 */
-                auto const phiReal = float_T(math::abs(phi));
-                float_T sinPhiReal;
-                float_T cosPhiReal;
-                pmacc::math::sincos(phiReal, sinPhiReal, cosPhiReal);
-                float_T const alphaTilt = math::atan2(float_T(1.0) - beta0 * cosPhiReal, beta0 * sinPhiReal);
-                /* Definition of the laser pulse front tilt angle for the laser field below.
-                 *
-                 * For beta0=1.0, this is equivalent to our standard definition. Question: Why is the
-                 * local "phi_T" not equal in value to the object member "phiReal" or "phi"?
-                 * Because the standard TWTS pulse is defined for beta0 = 1.0 and in the coordinate-system
-                 * of the TWTS model phi is responsible for pulse front tilt and dispersion only. Hence
-                 * the dispersion will (although physically correct) be slightly off the ideal TWTS
-                 * pulse for beta0 != 1.0. This only shows that this TWTS pulse is primarily designed for
-                 * scenarios close to beta0 = 1.
-                 */
-                float_T const phiT = float_T(2.0) * alphaTilt;
-
-                /* Angle between the laser pulse front and the y-axis. Not used, but remains in code for
-                 * documentation purposes.
-                 * float_T const eta = float_T(PI/2) - (phiReal - alphaTilt);
+                 * y-axis of the coordinate system in this function.
                  */
+                auto const phiT = float_T(math::abs(phi));
+                float_T sinPhi;
+                float_T cosPhi;
+                pmacc::math::sincos(phiT, sinPhi, cosPhi);
```

Comment:
Sorry for my ignorance but could you please quickly explain if `math` and `pmacc::math` are different namespaces. Also, given that those are frequently used, a `using` directive might be beneficial for this function.

Reply:
During the development of this code, namespaces have changed. I guess that this is a remainder that can be removed now. I'll check.

Reply:
This is still inconsistent but it's not blocking for me.